core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -324,6 +325,10 @@ protected[core] object WhiskWebActionsApi extends Directives {
     findContentTypeInHeader(headers, transid, `text/html`).flatMap { mediaType =>
       val ct = ContentType(mediaType, () => HttpCharsets.`UTF-8`)
       ct match {
+        // TODO: remove this extract check for base64 on json response
```

Comment:
Extra and break typos. Anyway not worth fixing should work to remove this with X day notice.